docs/operate/get-started/setup.md

```diff
@@ -76,26 +67,22 @@ If your SBC or other computer already has a supported operating system installed
 
 {{< /expand >}}
 
+1. Make sure your computer or SBC is powered on and connected to the internet.
+
 1. Create a [Viam app](https://app.viam.com) account.
-   The Viam app is the online hub for configuring and managing devices as well as viewing data.
+   The Viam app is the online hub for configuring and managing devices and data.
+
 1. Add a new _{{< glossary_tooltip term_id="machine" text="machine" >}}_ using the button in the top right corner of the **LOCATIONS** tab in the app.
    A machine represents your device.
-1. From your machine's page in the Viam app, follow the setup instructions to install `viam-server` or `viam-micro-server` on your device and connect it to the cloud.
-   [`viam-server`](/operate/reference/viam-server/) is the executable binary that runs on your device and manages hardware drivers, software, and data capture and sync.
-   [`viam-micro-server`](/operate/reference/viam-micro-server/) is the lightweight version of `viam-server` for microcontrollers.
-1. Use the **+** button on your machine's **CONFIGURE** tab to add [supported hardware components](/operate/get-started/supported-hardware/) so that `viam-server` can control your specific hardware.
-1. Use this same **+** button to configure software services such as [data capture and sync](/data-ai/capture-data/capture-sync/).
-
-As soon as you configure each component and save the configuration, you can use the **TEST** panel of the component's config card to, for example, view your camera's stream or turn your motor.
 
-## Concepts
+1. From your machine's page in the Viam app, follow the setup instructions to install `viam-server` on your device and connect it to the cloud.
 
-### What is a machine?
+1. A secure connection is automatically established between your machine and the Viam app.
+   When you update your machine's configuration, `viam-server` automatically gets the updates.
 
-A _machine_ is a computer (often a single-board computer like a Raspberry Pi or Jetson) or microcontroller and all the hardware attached to it, as well as the software running on it.
-You can think of one machine as representing one device, or one robot.
+   [You are ready to configure your machine's hardware ->](/operate/get-started/supported-hardware/)
```

Comment:
Similar to on "Learn the basics", could we make the link text match the left nav item?

Reply:
```suggestion
   You are ready to [configure supported hardware](/operate/get-started/supported-hardware/) on your machine.
```